### DISCUSSION TITLE
Error: pkg-config: Invalid preset `pour_bottle?` condition

### DISCUSSION THREAD

**Discussion by YIKAILucas** [Homebrew/discussions#2965] [Everyday usage]
I met this problem when I typed brew install xxx. the uninstall feature still work, but I cannot install it back and install any lib again

10.15 MacOS Catalina

**Comment by carlocab** [comment]
What is the output of `brew config` and `brew doctor`?

**Reply by YIKAILucas** [reply]
![This](https://vimeracke.oss-cn-shenzhen.aliyuncs.com/uPic/2022-02-17%2020:30:11%201645101011%20rEfrFn.png)

![2022-02-1720:33:101645101190IGahH1](https://vimeracke.oss-cn-shenzhen.aliyuncs.com/uPic/2022-02-17%2020:33:10%201645101190%20IGahH1.png)

**Reply by YIKAILucas** [reply]
The two pics above are `brew config` and `brew doctor`. And I tried to `brew install luajit-openresty`, It doesn't work

**Reply by SMillerDev** [reply]
Please don't send this kind of information as images, it's the absolute worst way to relay a text based output.

**Reply by SMillerDev** [reply]
You're working with a very old Homebrew version (9 months). Do `brew update`, and if that doesn't work run `brew update-reset` to update it before trying again.

**Reply by YIKAILucas** [reply]
Ok, thx. I'll try and show my terminal display in a better way

**Comment by Luuuxury** [comment]
VPN 开全局，重新 install
